docs/v5/order/create-order.mdx

```diff
@@ -158,6 +158,24 @@ print(session.place_order(
 ))
 ```
 
+</TabItem>
+<TabItem value="Go" label="Go">
+
+```go
+import (
+    "context"
+	"fmt"
+	bybit "github.com/wuhewuhe/bybit.go.api")
+client := bybit.NewBybitHttpClient("YOUR_API_KEY", "YOUR_API_SECRET", bybit.WithBaseURL(bybit.TESTNET))
+params := map[string]interface{}{"category": "linear", "symbol": "BTCUSDT", "side": "Buy", "positionIdx": 0, "orderType": "Limit", "qty": "0.001", "price": "10000", "timeInForce": "GTC"}
```

Comment:
Please let each key have its own line to stop it from being so long

Reply:
done